draft-ietf-quic-transport.md

```diff
@@ -618,10 +618,11 @@ any version negotiation that occurred (see {{version-negotiation}}).  The client
 MAY also retain any observed RTT or congestion state that it has accumulated for
 the flow, but other transport state MUST be discarded.
 
-The payload of the Server Stateless Retry packet contains STREAM frames and
-could contain PADDING and ACK frames.  A server can only send a single Server
-Stateless Retry packet in response to each Client Initial packet that is
-receives.
+The payload of the Server Stateless Retry packet contains a single
+STREAM frame on stream 0 with offset 0 containing the server's
+cryptographic stateless retry material. It MUST NOT contain any other
+frames. Any future stream frames from the server will also start at
+stream offset 0.
```

Comment:
Nit: STREAM frames.  Also,  this somewhat implies that ALL STREAM frames will have an offset of 0.  I think that you mean that the NEXT STREAM frame starts at an offset of 0.